app/main.py

```diff
@@ -1,8 +1,79 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+        self, comfort_class: int, clean_mark: int, brand: str
```

Comment:
When you have multiple params it is a best practice to have each parameter on new line